plugins/irma-server/README.md

```diff
@@ -71,6 +77,8 @@ session: {
     qrFromResult: r => r.sessionPtr
   },
 
+  handle: false,
```

Comment:
Is `handle` een handige naam voor deze property..? Het zegt me helemaal niks, en ik heb nu geen idee hoe ik deze feature moet gebruiken, ook niet na het lezen van de instructie hierboven. Wat verwacht `handle`? Een object? Een boolean? Een string? 😉

Reply:
`handle` verwacht een `sessionPtr` object zoals dat geleverd wordt door `startSession`  van de [IRMA server REST API](https://irma.app/docs/api-irma-server/#post-session) of zoals in [irma-backend-packages](https://github.com/privacybydesign/irma-backend-packages). Dit komt dus precies overeen met wat `handleSession` van de huidige `irmajs` als argument ontvangt. De naamconventie heb ik dus overgenomen vanuit de oude library. Dit zal ook voornamelijk de use case zijn waarin mensen het willen gebruiken. Nieuwe gebruikers zullen toch meer voor de `start` optie gaan, vandaar dat ik ervoor gekozen had om de naamconventie uit `irmajs` over te nemen.